web/src/main/webapp/css/variables.less

```diff
@@ -19,7 +19,15 @@
 // VARIABLES
 @white: #fff;
 @black: #000;
-@category-blue: #066999;
+
+/* Z-indices */
+@home-view-1: 59;
+
+/* Media queries */
+@xs: 600px;
+@sm: 960px;
+@md: 1280px;
+@lg: 1920px;
```

Comment:
:+1:

Reply:
Do these need to be in home to use them? I'd think we'd want to define these in the framework so all frame apps would be consistent?

Reply:
Ah, yeah. They actually are already in the framework. I suppose I didn't need them here.